install-steps.md

```diff
@@ -511,6 +511,35 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
+## Retrieving the OpenShift Installer (GA Release)
+
+The `latest-4.x` (i.e. `latest-4.3`) may be used to deploy the latest 
+Generally Available version of Red Hat OpenShift Platform.
+
+Configure VARS
+    ~~~sh
+    export VERSION=latest-4.3
+    export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs)
+    ~~~
+ 
+
+### Extract the Installer (GA Release)
+
+Once the installer has been chosen, the next step is to extract it. 
+
+~~~sh
+export cmd=openshift-baremetal-install
+export pullsecret_file=~/pull-secret.txt
+export extract_dir=$(pwd)
+# Get the oc binary
+curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/openshift-client-linux-$VERSION.tar.gz | tar zxvf - oc
+sudo cp ./oc /usr/local/bin/oc
```

Comment:
I'd say to use `sudo mv` to avoid having the binary messing around

Reply:
Not sure what you mean here. Currently Ansible using the copy module to the above and works fine.